pkg/florist/user_test.go

```diff
@@ -30,3 +32,20 @@ func TestUserSystemAddSuccessVM(t *testing.T) {
 
 	assert.NoError(t, err, "florist.UserSystemAdd")
 }
+
+func TestUserSystemAddWithUIDSuccessVM(t *testing.T) {
+	florist.SkipIfNotDisposableHost(t)
+
+	userName := "giuseppe"
+	userUid := 1035
```

Comment:
Although this value seems to work, it is a bad example, it should be below `1000`.
From https://man7.org/linux/man-pages/man5/login.defs.5.html

```
SYS_UID_MAX (number), SYS_UID_MIN (number)
Range of user IDs used for the creation of system users by useradd or newusers.

The default value for SYS_UID_MIN (resp.  SYS_UID_MAX) is 101 (resp.  UID_MIN-1).
```

```
UID_MAX (number), UID_MIN (number)
Range of user IDs used for the creation of regular users by useradd or newusers.

The default value for UID_MIN (resp.  UID_MAX) is 1000 (resp. 60000).
```

From all this, we can deduce `SYS_UID_MAX = 1000-1`

Reply:
So given that the artifactory user MUST be 1035... Then I should not create it as a system user but as a normal user? Which is strange, because normal user should be used by humans, while system user should be used to run specific services..

So how should I proceed?

Reply:
We have to drink the Kool Aid :-( but at least we do not want to give bad habits in the test value.
For me in this PR it is enough to use say 999 in the test value.
For Artifactory, we can discuss more if you want, but not here :-)

Reply:
> For me in this PR it is enough to use say 999

We are not that lucky: `adduser: The GID 999 is already in use.`
> sudo cat /etc/group
> systemd-journal:x:999:

It works fine with 899 though :)